config/initializers/schedule_parameters.rb

```diff
@@ -0,0 +1,6 @@
+#sanitize the OSEM_SCHEUDLE_CELL_SIZE to be used for EventType::LENGTH_STEP
```

Comment:
But could it be in the EventType model instead of in this new file?

We are modifying there the `SCHEDULE_CELL_SIZE` variable anyway: https://github.com/openSUSE/osem/pull/1249/files#diff-490f3d30636bbfbfaf4b4489e64bd430R20